test/Reachability/unit_solve_hybrid_bouncing_ball.jl

```diff
@@ -31,11 +31,13 @@ HS = HybridSystem(a, m, r, s);
 # initial condition in mode 1
 X0 = Hyperrectangle(low=[10, 0.0], high=[10.2, 0.0]);
 
+X_loc1 = Hyperrectangle(low=[9.8, 0.0], high=[10, 0.0]);
```

Comment:
Why is this set different from `X0`?

Reply:
just played around to make sure that it applies new X_loc1